dev/TabView/TabViewItem.cpp

```diff
@@ -79,3 +90,24 @@ void TabViewItem::OnIsCloseablePropertyChanged(const winrt::DependencyPropertyCh
 {
     UpdateCloseButton();
 }
+
+void TabViewItem::OnHeaderPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
+{
+    auto toolTipContent = winrt::ToolTipService::GetToolTip(*this);
+    if (!toolTipContent)
```

Comment:
This will make it so that once Header has a value and we set the tool tip accordingly, it'll then never change, which goes counter to what I interpret as your intention, which appears to be to have the tool tip always equal to the value of Header whenever the developer hasn't explicitly set one.  I would recommend having a bool that keeps track of whether or not the developer has set a value for ToolTip, and checking that instead of checking whether we have a tool tip.  You can do that by setting that bool to true when the value of ToolTip changes through a code path external to something in TabViewItem while ignoring expected changes to ToolTip (e.g., the setting occurring later in this method).

Reply:
Fair point. After some soul searching, here's my update... when the control first loads, I check if there's a custom tooltip, and if there isn't I create my own and then I update THAT one from there on out. If the app overrides it after the control is created, theirs will win out. I added two tests to verify that it's working as you'd expect now.

Reply:
That works too. :)